src/seedu/addressbook/ui/TextUi.java

```diff
@@ -162,7 +150,7 @@ private static String getIndexedListForViewing(List<String> listItems) {
      * @param visibleIndex visible index for this listing
      */
     private static String getIndexedListItem(int visibleIndex, String listItem) {
-        return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
+        return String.format(Formatter.MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
```

Comment:
This piece of code is doing formatting.
Could this be abstracted into `Formatter` class?